testing/events.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	"github.com/stretchr/testify/suite"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/cometbft/cometbft/abci/types"
```

Comment:
seems like we have slight inconsistency between `abci` alias and `abcitypes` alias. I think I prefer `abci` just because there is no go file at that level https://github.com/cometbft/cometbft/tree/main/abci, but I don't have a strong preference. We can make consistent in a followup